frontend/components/ui/blocknote/blocks/quiz-open-ended-block.tsx

```
@@ -0,0 +1,139 @@
+          props: { correctAnswer: e.target.value },
+        });
+      };
+
+      const handleDelete = () => {
+        props.editor.removeBlocks([props.block]);
+      };
+
+      return (
+        <div
+          ref={blockRef}
+          onMouseDown={(e) => {
```

ESLint

[error] 'e' is defined but never used.

(@typescript-eslint/no-unused-vars)